src/components/ui/form/GenericFormElement.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts" setup>
+import { computed } from 'vue';
+
+const props = defineProps<{
+  id: string;
+  name: string;
+  description?: string;
+  issues?: string[];
+}>();
+
+const inputClasses = computed(() => {
+  const classes = `bg-gray-50 border  text-sm rounded-lg block w-full p-2.5`.split(' ');
```

Comment:
Between `border`and `text-sm` there are two spaces. Is that on purpose?

Reply:
fixed